deployment/terraform/info.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package terraform
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// Info displays information about the current load-test deployment.
+func (t *Terraform) Info() error {
+	output, err := t.getOutput()
+	if err != nil {
+		return err
+	}
+
+	t.displayInfo(output)
+
+	return nil
+}
+
+func (t *Terraform) displayInfo(output *terraformOutput) {
+	fmt.Println("==================================================")
+	fmt.Println("Deployment complete 🎉. Here is the setup information:")
```

Comment:
We shouldn't be so sure about if the deployments is complete :)

Reply:
Fair enough. It's just that this command is shared between create and info. We need to separate the "Deployment complete" part.

Reply:
Also I wouldn't count on unicode support :grinning:

Reply:
Lol .. there goes me trying to be "modern". Back to the stone ages then.

(In all seriousness though, I would love to get some data on how many shells are there which don't support unicode).

Reply:
the true geek uses ASCII :)

Reply:
I like it when Agniva uses emojis 😂

Reply:
This is me trying to change myself.

Reply:
done